README.md

```diff
@@ -65,6 +65,31 @@ Currently the waiting period is as follows:
 
 ## FAQ
 
+#### Do I need to register a package to install it?
+
+No, you can simply do `using Pkg; Pkg.add(url="https://github.com/JuliaLang/Example.jl")`
+to e.g. install the package `Example.jl`, even if it was not registered.
```

Comment:
It is worth saying here that if you register it people will be able to install it via `pkg"add Example"`.
And maybe that they will be able to install it  by activating a Project.toml that has it listed and running `Pkg.resolve()`
(in contrast to if it was installed via URL, in which case they would need a whole Manifest.toml)

Reply:
I expanded on this section a bit now